active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,184 @@
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
+```
+
+![](images/numberbox-basic.png)
+
+### Add placeholder text
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+![](images/numberbox-placeholder-text.png)
+
+### Enable calculation support
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+![](images/numberbox-acceptscalculation.png)
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for items in order" 
+    StepSize="0.1"
+    UpDownButtonsEnabled="True"
+    UpDownDragEnabled="True" />
+```
+
+![](images/numberbox-updownbuttons.png)
+
+### Add mininum and maximum values
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="NumberBox for items in order" 
+    PlaceholderText="1" 
+    MinValue="1"
+    MaxValue="99" />
+```
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.TextBox
+{
+    NumberBox();
+    
+    Double Value;
+    
+    Boolean AcceptsCalculation;
+    Boolean UpDownButtonsEnabled;
+    Boolean HyperDragEnabled;
+    Boolean HyperScrollEnabled;
+    
+    Double StepSize;
+  
+    Boolean AreLeadingZerosTrimmed;
+    Double DecimalPrecision;
+    Boolean DoesInputRound;
+    
+    Double MinValue;
+    Double MaxValue;
+    
+
+    static Windows.UI.Xaml.DependencyProperty ValueProperty{ get; };
```

Comment:
Can I correctly assume that the way this is displayed in the UI will be formatted based on OS locale settings?

Reply:
That is correct. We'll be leveraging that capability with the intention to avoid requiring the developer needing to handle localization.